tests/test_nhshealthierfamilies_2.py

```diff
@@ -0,0 +1,83 @@
+from recipe_scrapers._grouping_utils import IngredientGroup
+from recipe_scrapers.nhshealthierfamilies import NHSHealthierFamilies
+from tests import ScraperTest
+
+
+class TestNHSHealthierFamiliesScraper(ScraperTest):
+
+    scraper_class = NHSHealthierFamilies
+    test_file_name = "nhshealthierfamilies_2"
+
+    def test_host(self):
+        self.assertEqual("nhs.uk", self.harvester_class.host())
+
+    def test_author(self):
+        self.assertEqual("NHS Better Health", self.harvester_class.author())
+
+    def test_title(self):
+        self.assertEqual("Good old fish and chips", self.harvester_class.title())
+
+    def test_total_time(self):
+        self.assertEqual(50, self.harvester_class.total_time())
+
+    def test_yields(self):
+        self.assertEqual("4 servings", self.harvester_class.yields())
+
+    def test_image(self):
+        self.assertEqual(
+            "https://assets.nhs.uk/campaigns-cms-prod/images/Recipes-square-healthy-fish-and-chips.width-320.jpg",
+            self.harvester_class.image(),
+        )
+
+    def test_ingredients(self):
+        expected_ingredients = [
+            "4 potatoes, scrubbed, each cut into 8 wedges",
+            "1 tablespoon vegetable oil",
+            "75g dried white or wholemeal breadcrumbs",
+            "1 egg, beaten with 2 tbsp cold water",
+            "4 fillets skinless white fish, like haddock, cod or pollock",
+            "300g mushy peas",
+            "1 pinch ground black pepper (optional)",
+        ]
+        self.assertEqual(expected_ingredients, self.harvester_class.ingredients())
+
+    def test_ingredient_groups(self):
+        self.assertEqual(
+            [
+                IngredientGroup(
+                    ingredients=[
+                        "4 potatoes, scrubbed, each cut into 8 wedges",
+                        "1 tablespoon vegetable oil",
+                        "75g dried white or wholemeal breadcrumbs",
+                        "1 egg, beaten with 2 tbsp cold water",
+                        "4 fillets skinless white fish, like haddock, cod or pollock",
+                    ],
+                    purpose=None,
+                ),
+                IngredientGroup(
+                    ingredients=[
+                        "300g mushy peas",
+                        "1 pinch ground black pepper (optional)",
+                    ],
+                    purpose="Swappable or optional",
```

Comment:
Kind of a nitpick/philosophical thought: in most recipes the groupings are for subcomponents of the meal used during preparation/cooking.  In this case, this grouping is for some extra items that are each individually nice-to-haves.

I think it's fine - in fact I think ingredient groups are a good solution here.  It's mostly an observation that I wanted to mention.

Reply:
I made sure to check against another recipe that was more inline with the actual purpose and everything works. I guess this is just a new use-case for it!

https://www.nhs.uk/healthier-families/recipes/bengali-chicken-curry/

Reply:
Neat :)  thanks!